src/main/cpp/utils/logging/Logger.h

```diff
@@ -69,6 +69,9 @@ class Logger
     /// @brief Read logging option from dashboard, but not every 20ms
     void PeriodicLog();
 
+    /// @brief Logs some motor data directly to the network tables, so that the rest of logging does not need to be enabled
+    void LogDataDirectlyOverNT(const std::string &group, const std::string &identifier, double value) const;
```

Comment:
Maybe overload this with booleans and strings to support digital inputs and state names instead of IDs...